### DISCUSSION TITLE
SERIALNUMBER vs SN

### DISCUSSION THREAD

**Discussion by 8805102** [Keyfactor/ejbca-ce#381] [Q&A]
in my end entity profile I've selected "serialNumber, Serial number (in DN)", it shows up in the certificate as **SN=xxxxx**
How do I get it to show up as **SERIALNUMBER=xxxx**

thanks,
mjk

**Comment by primetomas** [comment]
This is a feature of whatever application you use to display it with. Internally in the certificate it's not encoded as a string like that but an object identifier. So whatever you see as a human readable string is a feature of your certificate viewer. It doesn't matter how it's displayed in your viewer, the value in the actual certificate is the same, even if it displays differently in two different applications.

**Reply by 8805102** [reply]
Thanks.  Should have realized that as a possibility.

mjk
